src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -90,6 +90,8 @@ def load_arguments(self, _):
         c.argument('cluster_name', options_list=['--name', '-n'], id_part='name', help='The name of the connected cluster.')
         c.argument('kube_config', options_list=['--kube-config'], help='Path to the kube config file.')
         c.argument('kube_context', options_list=['--kube-context'], help='Kubconfig context from current machine.')
+        c.argument('force_delete', options_list=['--force'], help='Force delete to remove all hanging azure-arc resources from the cluster even if the config map is missing.', action='store_true')
+        c.argument('yes', options_list=['--yes', '-y'], help='Do not prompt for confirmation.', action='store_true')
```

Comment:
@anraghun to review the CX facing help text

Reply:
Force delete option to remove all ~hanging~ azure-arc resources from the cluster ~even if the config map is missing~.